src/entities/Profile.ts

```diff
@@ -21,6 +21,16 @@ export enum Course {
 	SOFTWARE_ENGINEERING = 'Software Engineering'
 }
 
+export enum Year {
+	ONE = 'First Year',
+	TWO = 'Second Year',
+	THREE = 'Final Year Bachelors',
+	FOUNDATION = 'Foundation Year',
+	MASTERS = 'Masters Year',
+	INDUSTRIAL = 'Industrial Year',
+	PHD = 'PHD Student'
```

Comment:
Bit pedantic, but could we lowercase the `h` in PhD?
```suggestion
	PHD = 'PhD Student'
```